.builds/freebsd.yml

```diff
@@ -0,0 +1,30 @@
+image: freebsd/latest
+packages:
+  - devel/cmake
+  - devel/pkgconf
+  - lang/python3
+  - print/freetype2
+  - x11-fonts/fontconfig
+  - x11-fonts/dejavu
+  - x11/libxcb
+sources:
+  - https://github.com/alacritty/alacritty
+tasks:
+  - rustup: |
+      curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable --profile minimal
+  - 1-39-0: |
+      $HOME/.cargo/bin/rustup toolchain install --profile minimal 1.39.0
+      cd alacritty
+      $HOME/.cargo/bin/cargo +1.39.0 test
+  - stable: |
+      $HOME/.cargo/bin/rustup toolchain install --profile minimal stable
+      cd alacritty
+      $HOME/.cargo/bin/cargo +stable test
+  - stable-clippy: |
+      $HOME/.cargo/bin/rustup component add --toolchain stable clippy || true
+      cd alacritty || true
+      $HOME/.cargo/bin/cargo +stable clippy || true
```

Comment:
Is the `|| true` to make this stage 'optional'?

I feel like at that point this stage is entirely useless, because we get no feedback at all about its success or failure.

Reply:
Yeah, it doesn't support 'Allowed failures' feature. We can remove 'nightly' and 'clippy' for now, I guess.